pandas/meson.build

```diff
@@ -0,0 +1,51 @@
+incdir_numpy = run_command(py,
```

Comment:
Is there nothing built in to meson that handles this natively?

Reply:
Sadly no, mesonbuild/meson#9598 tracks this.

TLDR: it probably requires some changes on the numpy side for meson to auto-detect numpy.